man/git-reauthor.md

```diff
@@ -0,0 +1,83 @@
+git-reauthor(1) -- Rewrite history to change author's identity
+==============================================================
+
+## SYNOPSIS
+
+`git reauthor [<options>]`
+
+## DESCRIPTION
+
+Rewrite history to change author's identity.
```

Comment:
I agree. I did it like that because that's the way I found it done in many commands in the repo.
removed in commit 68fb3d5